CHANGELOG.md

```diff
@@ -6,6 +6,16 @@
 
 ### Fixes
 
+## v1.19.0 (Dec 1, 2021)
+
+### Enhancements
+
+- Create a user interface to identify interesting traces (high qps, slow or erroneous) without knowing the service or operations up-front ([@th3M1ke](https://github.com/th3M1ke) in [#815](https://github.com/jaegertracing/jaeger-ui/pull/815)). See [#2954](https://github.com/jaegertracing/jaeger/issues/2954) for more details.
```

Comment:
The description feels very cryptic. What does "without knowing the service or operations up-front" have to do with the functionality?

Can we try to make it sound more interesting? Even the ticket title would be better: "Monitor" tab for service health metrics